plugins/build-your-own.rst

```diff
@@ -25,13 +25,9 @@ This interface defines the ``handleRequest`` method that allows to modify behavi
      */
     public function handleRequest(RequestInterface $request, callable $next, callable $first);
 
-The ``$request`` comes from an upstream plugin or Plugin Client itself.
+The ``$request`` comes from an upstream plugin or ``PluginClient`` itself.
 You can replace it and pass a new version downstream if you need.
 
-.. note::
-
-    Be aware that the request is immutable.
```

Comment:
I guess it is just a gental reminder that you can to: 

```php
$request->withAddedHeader('Foo', 'Bar');
```

Im okey with removing it.